scripts/s3manage.py

```diff
@@ -338,64 +331,52 @@ def delete_version(s3_path, bucket_name):
         n = 200
         indexes = [{'Key': k.key} for k in files]
         for i in xrange(0, len(indexes), n):
-            resp = s3client.delete_objects(Bucket=bucket_name, Delete={'Objects': indexes[i: i + n]})
+            resp = s3client.delete_objects(
+                Bucket=bucket_name, Delete={
+                    'Objects': indexes[
+                        i: i + n]})
             for v in resp['Deleted']:
                 print(v)
     else:
         print('Aborting deletion of <%s>.' % s3_path)
 
 
-def get_url(deploy_target, key_name='index.html'):
-    bucket_host = 'mf-geoadmin3.%s.bgdi.ch' % deploy_target
-    object_url = 'https://%s/%s' % (bucket_host, key_name)
-    return object_url
-
-
 def activate_version(s3_path, bucket_name, deploy_target):
-    if version_exists(s3_path) is False:
-        print('Version <%s> does not exists in AWS S3. Aborting' % s3_path)
-        sys.exit(1)
-
-    msg = raw_input('Are you sure you want to activate version <%s>?\n' % s3_path)
```

Comment:
@procrastinatio the confirmation request is completely removed, am I right?

Reply:
No, I moved it in the `activate_cmd` function and add a `--force` parameter for future Jenkins use :-)

Reply:
Same, for `upload` you have to confirm the target and path (also a `--force` for jenkins)